### DISCUSSION TITLE
change date format

### DISCUSSION THREAD

**Discussion by StefanoCecere** [vuepress-theme-hope/vuepress-theme-hope#4084] [Q&A]
it seems difficult to change the date format to dd/mm/yyyy or even better dd month yyyy
Vuepress doesn't seem to have this feature native, right?
i did find an answer online (with standard Vuepress) and the answer was to use a custom JS function!!

how is the situation? can't it be a simple "format string" as in all projects?

**Comment by Mister-Hope** [comment]
cc @meteorlxy
